tests/unit/models/test_food_analysis.py

```
@@ -0,0 +1,172 @@
+    """Tests for the Ingredient model."""
+    
+    def test_ingredient_creation(self):
+        """Test creating an ingredient."""
+        ingredient = Ingredient(name="Test Ingredient", servings=100)
+        assert ingredient.name == "Test Ingredient"
+        assert ingredient.servings == 100
+    
+    def test_ingredient_validation(self):
+        """Test ingredient validation."""
+        # Missing name should raise error
+        with pytest.raises(Exception):
```

Ruff (0.8.2)

`pytest.raises(Exception)` should be considered evil

(B017)